setup.py

```diff
@@ -83,6 +83,8 @@ def pre_post_plugin_type():
     requirements = ["netifaces", "aexpect", "netaddr", "simplejson", "six"]
     if sys.version_info[:2] >= (3, 0):
         requirements.append("avocado-framework>=68.0")
+        if sys.version_info[:2] >= (3, 3):
+            os.system("rm -rf virttest/passfd*")
```

Comment:
I'd discourage everyone from removing files that are in git. By simply executing any `setup.py` command this leads to a change on user filesystem.